pkg/acsengine/defaults.go

```diff
@@ -158,6 +158,7 @@ func setMasterNetworkDefaults(a *api.Properties) {
 				// When VNET integration is enabled, all masters, agents and pods share the same large subnet.
 				a.MasterProfile.Subnet = a.OrchestratorProfile.KubernetesConfig.ClusterSubnet
 				a.MasterProfile.FirstConsecutiveStaticIP = getFirstConsecutiveStaticIPAddress(a.MasterProfile.Subnet)
+				a.MasterProfile.VnetCidr = (a.MasterProfile.VnetCidr)
```

Comment:
@jackfrancis is this required given that I set a default for the param in kubernetesparams.t